test/async-hooks/test-get-connections.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const assert = require('assert');
+const net = require('net');
+const server = net.createServer();
+
+try {
+    server.getConnections(() => {
+        assert.ok(true, "No error thrown")
```

Comment:
You can just call `server.getConnections(common.mustCall())` or `server.getConnections(common.mustCall(fn))`, that’s a bit shorter (and actually makes sure that the callback gets executed) :)